app/lambda/create_session/lambda_function.py

```diff
@@ -60,6 +57,6 @@ def handle(self, event, context):
                 }
             }
         except Exception as e:
-            print(str(e))
+            print("Unexpected error: %s" % e)
```

Comment:
During error handline, we need to return the right error code in Lambda API

Reply:
Added.